transaction-flow.md

```diff
@@ -200,6 +202,20 @@ If a sender has been assigned an `external_id`, this value can be used to find s
   * If you already do KYC on your system, then please contact us and we can waive this requirement from you. In this case you should send us an empty list of documents: `"documents": [ ]` in the request. All of the senders you create in the system will be immediately set to the `approved` state and you won't need to wait for them to get approved.
   * If when creating senders or transactions you get the following error in the response: `"errors":{"documents":[{"error":"blank"}]}` it means that KYC requirements are not yet waived for your account. If we already approved your KYC process and so they should be, then please contact us so we can rectify the issue and update your account accordingly.
   * In case you don't do KYC on your site, then you will need to send us documents that we can use to verify the sender's identity, for more details on this please see the [API reference documentation](https://api.transferzero.com/documentation#documents).
+* If you do not wish to use Cashplus or MAD:Cash as corridor, then we can enable WTR2 rules for you to identify the `sender`.
```

Comment:
We should start with something like: `We also support sender validation based on WTR2 rules`, then the details on what WTR2 is should be indented into these - so it's a separate block. Notes like cashplus not working should be in this indent. Also make sure to format `MAD::Cash` appropriately, and use block characters